app/views/reports/report.pdf.erb

```diff
@@ -9,7 +9,8 @@
   </head>
   <body class="print-report-body">
     <div class="print-report">
-      <%= sanitize_input(@html) %>
+      <% # Also whitelist <img> and <input type="checkbox"> tags %>
+      <%= sanitize_input(@html, ['img', 'input'], ['type', 'disabled', 'checked']) %>
```

Comment:
Is there a reason, why you didn't whitelist the tags and attributes directly in constants.rb file?